src/command_panel.nim

```diff
@@ -0,0 +1,38 @@
+import strutils
+import typetraits
+
+proc commandPanel*(): tuple[y: int, x: int] =
```

Comment:
Overall this is really great work man. I personally think that code that includes more than one continue in a while loop is doing more than it should, but this is a prototype, and this is more than enough to get the job done here.

In the future, let's avoid long procedures and break it up if the logic to solve a problem requires more than a few statements.